src/components/ReportButton.tsx

```diff
@@ -7,17 +7,19 @@ import { DangerIcon } from '.';
 export type ReportButtonProps = {
   onClick: () => void;
   reported: boolean;
+  size?: 'sm' | 'md' | 'lg';
```

Comment:
added to support make customise the Report button size, current version is very small to show in snap slug screen